README.md

```diff
@@ -150,23 +150,61 @@ docker stop {CONTAINER}
 eg docker stop frosty_bartik
 ```
 
-# Module 6 - Architecture Diagrams
+# Architecture Diagrams (Module 6)
 
 See the documentation folder
 
 The diagrams are saved as .jpg files, named per level of The C4 Model - as per https://c4model.com/ 
 
 The source code for these is in same named .drawio files and can be edited using https://app.diagrams.net/
 
-# Module 7 - Continuous Integration
+# Continuous Integration (Module 7)
 
 ## To run the tests from docker
 ```
 docker run --env-file ./.env.test todo-app:test
 ```
-## New github workflow file
+## github workflow file 
 
-This dictates the pipelines that run on GitHub to test every branch
+### ...was created to run the unit tests: 
+
+See the .yml file which stored at below location, which dictates the pipelines that run on GitHub to test every branch
+```
+.github/workflows/
+```
+# Continuous Delivery (Module 8)
+
+The GitHub actions workflow file ( .github/workflows/ci-and-cd-pipeline.yml ) has been enhanced to provide continuous deployment to **Heroku** hosting platform, using an image pushed to **Docker Hub**.
+This deployment is conditional. The default is for it to be conditonal upon:
+1) the git action is a pull or push
+2) project built and tests passed
+3) target is main branch or module8 branch
+```
+### Secrets
+Secrets have been stored in the **GitHub repository** to store credentials to facilitate:
+* Docker Hub signon
+* Heroku athentication - via an auth token
+* todo_app .env variables - which relate to the Trello data used in the app
+
+These secrets are only available to the GitHub repository owner, not to others who it is shared with.
+
+
+### URI to access the deployed app:
+  https://ann-todo-app.herokuapp.com 
```

Comment:
Thanks for all this clear documentation - apart from it's use in helping us remember things when we look back it also makes my life reviewing much easier!